main/src/com/google/refine/model/ReconStats.java

```diff
@@ -51,21 +51,26 @@ public class ReconStats {
     final public int newTopics;
     @JsonProperty("matchedTopics")
     final public int matchedTopics;
+    @JsonProperty("errorTopics")
+    final public int errorsTopics;
 
     @JsonCreator
     public ReconStats(
             @JsonProperty("nonBlanks") int nonBlanks,
             @JsonProperty("newTopics") int newTopics,
-            @JsonProperty("matchedTopics") int matchedTopics) {
+            @JsonProperty("matchedTopics") int matchedTopics,
+            @JsonProperty("errorsTopics") int errorsTopics) {
```

Comment:
To be consistent with the `@JsonProperty` annotation above:
```suggestion
            @JsonProperty("errorTopics") int errorsTopics) {
```